resources/timezones/README.md

```diff
@@ -0,0 +1,31 @@
+# Timezone Mapper
+
+The timezone mapper allows one to get a likely timezone for a particular phone
+number. For mobile phones which are associated with particular area codes, it
```

Comment:
We should mention that the timezone returned is the canonical ID from CLDR (and link to CLDR), not a localised name (or any other identifier) - and then below in the "contributing" part we can mention that IDs in CLDR don't change (or whatever Mark said that time we raised the bug).